packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -5,6 +5,15 @@ const cli = {
   describe: 'Run wdio tests',
   builder: (yargs) => (
     yargs.options({
+      assetServerPort: {
```

Comment:
`assetServerPort` is replacing [WDIO_INTERNAL_PORT](https://github.com/cerner/terra-toolkit/blob/main/packages/terra-functional-testing/src/config/wdio.conf.js#L24) which is used as the _port_ in AssetServerService. We currently already have a `port` CLI options so naming this `INTERNAL_PORT` seems confusing with the existing `port` option. So the name is changed to `assetServerPort`.